recipes/xcube-sh/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "xcube-sh" %}
+  {% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dcs4cop/xcube-sh/archive/{{ version }}.tar.gz
+  sha256: 7cef5ce7db192907ae6d41fc9b7198c359e5db4799c7b7dc22a0770befe13850
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install requests_oauthlib -vv && {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
```

Comment:
There are some differences here with the dependencies list in environment.yml (e.g. here psycopg2 is missing).